leaf-ui/js/displayAnalysis.js

```diff
@@ -272,12 +272,42 @@ function addFirstAnalysisConfig(){
     console.log(analysisMap)
 }
 
+/**
+ * Function to load analysis configurations and results from JSON into the config sidebar
+ * TODO: before running, wipe all other variables and analysis sidebars 
+ * so that we can load a new thing after running others
+ */
+function loadAnalysis(){
+    // Loop through each configuration
+	for(let config of analysisMap.values()) {
+        console.log(config);
+        currAnalysisConfig = config;
+        analysisRequest = currAnalysisConfig.getAnalysisRequest();
+        // TODO: Find better way to preserve original default from model
+        // Currently necessary for User Assignments List preservation
+        defaultUAL = currAnalysisConfig.userAssignmentsList;
```

Comment:
I don't think you need this here, its for perserving the User Assignments list when you originally load a model without configs. Which is why it was in the addFirstConfig. But I think since it should be in all the configs you load its not necessary to redo for all the configs. If you do need it I think it would only be once at the end outside the for loop, but I think you should be good dropping it completely.

Reply:
ok good to know -- I grabbed this from your code and didn't fully understand what it as doing. Thanks for the heads up!